### ISSUE TITLE
YAML schema is complaining about `pymdownx.magiclink`

### ISSUE THREAD

**Issue by Fullchee** [squidfunk/mkdocs-material#4081]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

In `mkdocs.yml`

`markdown_extensions -> pymdownx.magiclink` is showing an error highlight in VSCode

<img width="761" alt="image" src="https://user-images.githubusercontent.com/11246258/177001252-1967dd80-30c4-4c44-819e-af45a7fd6189.png">

I'm not sure why, given that it's in in the schema

https://squidfunk.github.io/mkdocs-material/schema/extensions/pymdownx.json

### Expected behaviour

It shouldn't be underlined with an error

### Actual behaviour

It is underlined with an error

### Steps to reproduce

1. Install VSCode
2. Install the YAML extension (by RedHat)
3. Add `pymdownx.magiclink` under `markdown_extensions` in `mkdocs.yml`

### Package versions

- Python 3.9.13
- mkdocs, version 1.3.0
- Name: mkdocs-material Version: 8.3.8

### Configuration

```yaml
markdown_extensions:
  - pymdownx.magiclink
```
```


### System information

OS: macOS Monterey 12.1 (M1)

VSCode: 1.68.1

**Comment by squidfunk** [comment]
Thanks for reporting. We have the definitions for `pymdowx.magiclink` here:

https://github.com/squidfunk/mkdocs-material/blob/4151f326dcfc2734eed62bc4f02b8ee415f3a429/docs/schema/extensions/pymdownx.json#L304

Works on my machine, i.e. cannot reproduce:

<img width="537" alt="Bildschirmfoto 2022-07-02 um 15 09 30" src="https://user-images.githubusercontent.com/932156/177002225-16b4a1f1-17f5-4e5e-9d69-3257baafd038.png">

**Comment by squidfunk** [comment]
I now see where this is coming from – we didn't include the attribute-less version of this extension. Fixed in adabdd0be.
